test/functional/ui/syntax_conceal_spec.lua

```diff
@@ -3,6 +3,14 @@ local Screen = require('test.functional.ui.screen')
 local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
 local insert = helpers.insert
 
+do
+  clear()
+  if helpers.os_name() == 'windows' then
+    pending('FIXME: Windows', function() end)
+    return
+  end
+end
```

Comment:
the `return` is still needed in the test file itself, unfortunately. Something like this maybe:

```lua
if helpers.pending_if_win32() then return end
```